google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UpdateFeedItemAttributeValue.java

```diff
@@ -244,7 +270,7 @@ private int getAttributeIndex(
 
     // Throws an exception if the attribute value is not found.
     if (attributeIndex == null) {
-      throw new RuntimeException("Invalid attribute index");
+      throw new IllegalArgumentException("Invalid attribute index");
```

Comment:
The invalid arg is the ID of the FeedItemAttributeValue, not the attribute index. Also, it's helpful to provide the illegal arg value in the message, e.g.,

`new IllegalArgumentException("No matching feed attribute for feed item attribute value: " + newFeedItemAttributeValue);`